### ISSUE TITLE
CQL: Example 15 needs editing

### ISSUE THREAD

**Issue by jratike80** [opengeospatial/ogcapi-features#486]
The example 15 in https://github.com/opengeospatial/ogcapi-features/blob/master/extensions/cql/examples/examples.txt needs some edits:
- geometry is propably in long/lat
- at least lat cannot be -118
- are coordinates flipped intentionally in CQL vs. JSON?

```
15. Location in the box between -118,33.8 and -117.9,34 in lat/long (geometry 1)

CQL: WITHIN(location,ENVELOPE(-118,33.8,-117.9,34)

CQL JSON: {
             "within" {
                "property": "location",
                "value": { "bbox": [33.8,-118,34,-117.9] }
              }
           }
```

**Comment by pvretano** [comment]
@jratike80 just a typo.  I fix! ;)

**Comment by cportele** [comment]
Fixed in #490
